app/router.js

```diff
@@ -1,21 +1,29 @@
 import EmberRouter from '@ember/routing/router';
 import config from 'frontend-worship-organizations/config/environment';
 
+import ENV from 'frontend-worship-organizations/config/environment';
 export default class Router extends EmberRouter {
   location = config.locationType;
   rootURL = config.rootURL;
 }
 
 Router.map(function () {
   this.route('login');
-  this.route('mock-login');
-  this.route('switch-login');
+  // CONTROLLER_LOGIN
+  if (ENV.controllerLogin !== 'true') {
+    this.route('mock-login');
+    this.route('switch-login');
+  } else {
+    this.route('controller-login');
+  }
 
   this.route('auth', { path: '/authorization' }, function () {
     this.route('callback');
     this.route('login');
     this.route('logout');
-    this.route('switch');
+    if (ENV.controllerLogin !== 'true') {
+      this.route('switch');
```

Comment:
it was not possible to add a switch to the controller login ?

Reply:
I don't think this will be feasible without adding custom code, but I don't see a major wall we may hit if we decide to implement it in the future. This is more like a MVP for now, assuming support will use the controle environment only for a specific organization for a specific problem. If it becomes too painful for them to logout/login all the times, we should definitely consider a switch.

Reply:
Perfect thanks for the answer !